astro/src/content/blog/fusionauth-reinvent-2024.mdx

```diff
@@ -0,0 +1,64 @@
+---
+publish_date: 2024-11-15 
+title: "FusionAuth at AWS re:Invent 2024"
```

Comment:
This uppercases the `r` in `re:Invent`. Don't know any way around it because it is a CSS issue. So I guess it is okay.